__tests__/server.test.js

```diff
@@ -44,3 +44,41 @@ describe("/api/topics", () => {
 		});
 	});
 });
+
+describe("/api/articles", () => {
+	describe("GET", () => {
+		describe("GET article by Id", () => {
+			test("GET 200, responds with an article when requested with article_id", () => {
+				return request(app)
+					.get("/api/articles/1")
+					.expect(200)
+					.then(({ body }) => {
+						expect(body).toHaveProperty("article_id", 1);
+						expect(body).toHaveProperty("title");
+						expect(body).toHaveProperty("topic");
+						expect(body).toHaveProperty("author");
+						expect(body).toHaveProperty("body");
+						expect(body).toHaveProperty("created_at");
+						expect(body).toHaveProperty("votes");
+						expect(body).toHaveProperty("article_img_url");
```

Comment:
As it's just a single object, a better test would also be to test the values instead of just the data types. `toEqual` would be very handy here!